File: margin/src/main/java/com/binance4j/margin/dto/Transaction.java
package com.binance4j.margin.dto;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

/**
 * A transfer, borrow or repay transaction.
 * 
 * @property tranId    Transaction id.
 * @property clientTag Client tag.
 */
@ApiModel("A transfer, borrow or repay transaction.")
data class Transaction(
@ApiModelProperty("Transaction id.")
var tranId:String?=null,
@ApiModelProperty("Client tag.")
var clientTag:String?=null)
{
}

File: connectors/src/main/kotlin/com/binance4j/margin/dto/Transaction.kt
package com.binance4j.margin.dto;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

/**
 * A transfer, borrow or repay transaction.
 * 
 * @property tranId    Transaction id.
 * @property clientTag Client tag.
 */
@ApiModel("A transfer, borrow or repay transaction.")
data class Transaction(
@ApiModelProperty("Transaction id.")
 val tranId : String,
@ApiModelProperty("Client tag.")
 val clientTag : String)
{
}